### PR TITLE
Added Documentation change on static files

### PR DESCRIPTION
**Pull request by pranjalg13**
## What does this PR do?

I updated the documentation on building static files and redirecting to the correct repo.

## Test Plan

NA

## Related PRs and Issues

https://github.com/appwrite/appwrite/issues/5318

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it include updated API specs and example docs?

### CODE REVIEW

**Review by ArmanNik — DISMISSED** [review]
Good catch! :blush: 
We might also want to link our [contribution guide](https://github.com/appwrite/console/blob/main/CONTRIBUTING.md) and invite people to contribute.

I would also mention that we use [Pink](https://github.com/appwrite/pink) for our CSS.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -109,7 +109,9 @@ docker run --rm --interactive --tty \
 
 ### User Interface
 
-Appwrite uses an internal micro-framework called Litespeed.js to build simple UI components in vanilla JS and [less](http://lesscss.org/) for compiling CSS code. To apply any of your changes to the UI, use the `gulp build` or `gulp less` commands, and restart the Appwrite main container to load the new static files to memory using `docker compose restart appwrite`.
+Appwrite UI is built with two frameworks [Svelte](https://svelte.dev/) and [Svelte Kit](https://kit.svelte.dev/). You can refer the repo [Appwrite-Console](https://github.com/appwrite/console) to learn and explore. Appwrite is using [Pink](https://github.com/appwrite/pink) for CSS.
```
Comment (gewenyu99) [inline CONTRIBUTING.md:112]:
```suggestion
Appwrite's UI is built with [Svelte](https://svelte.dev/), [Svelte Kit](https://kit.svelte.dev/), and the [Pink Design](https://github.com/appwrite/pink) component library. You can find the source code in the [Appwrite Console](https://github.com/appwrite/console) repository.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -109,7 +109,9 @@ docker run --rm --interactive --tty \
...
+
+To contribute to the UI, head over to the [Contribution Guide](https://github.com/appwrite/console/blob/main/CONTRIBUTING.md)
```
Comment (gewenyu99) [inline CONTRIBUTING.md:114]:
```suggestion
To contribute to the UI, head to the [Contribution Guide](https://github.com/appwrite/console/blob/main/CONTRIBUTING.md) of the Appwrite Console.
```

**Review by gewenyu99 — CHANGES_REQUESTED** [review]
Small comments :)

Thanks for catching this!

**Review by gewenyu99 — APPROVED** [review]

**Review by TorstenDittmann — APPROVED** [review]

**Comment by pranjalg13** [comment]
how is the test failing 😄 , is this because not synced with master? @gewenyu99 ?

**Comment by gewenyu99** [comment]
> how is the test failing 😄 , is this because not synced with master? @gewenyu99 ?

I wouldn't worry about it :P

**Pull request merged**